package.json

```diff
@@ -295,7 +296,9 @@
             "**/svgTransform.js"
           ]
         }
-      ]
+      ],
+      "no-shadow": "off",
+      "@typescript-eslint/no-shadow": "error"
```

Comment:
What does this do?

Reply:
This fixes a parsing bug in eslint for how typescript enums are treated.  It would think you were using the same variable name twice.  This disables the no-shadow rule in core eslint but then enables an alternative one in `@typescript-eslint`